File: Uni/C/Fibonacci.c
#include <stdio.h>

int main(){
	int counter, feins, fzwei;
	int i = 0;
	fzwei = 1;
	feins = 0;
	printf ("Bitte Zahl eingeben: ");
	scanf ("%d", &counter);
 
	int result;
	for (i=1; i< counter; i++){
		result = feins + fzwei;
		feins = fzwei;
		fzwei = result;
	}
	printf ("Die %dte Fibonacci Zahl ist: %d", counter, result);
}

File: Uni/Java/Java1-2/Fibonacci.java
public class Fibonacci{
public static void main(String[] args){

double f0=0;
double f1=1;
double f2=1;
int i=0;
int j=Integer.parseInt(args[0]);
double [] f = new double[j+3];
double q;

        System.out.println("Fibonacci-Zahl: " + "0.0");
        System.out.println("Fibonacci-Zahl: " + "1.0");

        f[0]=0.0;
        f[1]=1.0;

        for (i=0; i<j; i=i+1)
          {
          
          f2=f0 + f1;

          q=f1/f0;

          f[i+3]=f2;
          System.out.println("Fibonacci-Zahl: " + f2+" 		    Quotient: " + q);

          f0=f1;
          f1=f2;
          
          }

        }

}
